examples/javascript/src/locations/EntryList.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect } from 'react'
+import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
+
+const EntryList = () => {
+  const sdk = useSDK()
+
+  useEffect(() => {
+    const onEntryListUpdated = ({entries}) => {
+      const values = entries.reduce(
+        (res, item) => {
+          res[item.sys.id] = `${item.sys.id}`
+          return res
+        },
+        {}
+      )
+
+      return {
+        values,
+      }
+    }
+
+    sdk.entryList.onEntryListUpdated((props) => onEntryListUpdated({entries: props.entries}))
```

Comment:
Does `onEntryListUpdated` return an unsubscribe method? If yes, we should add a `return` here

Reply:
It doesn't return an unsubscribe method